### PR TITLE
fix transparent header and lightweight theme oddness

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

**Comment by bwinton** [comment]
Can I get a screenshot up in this place?

File: skin/base.css [hunk 1]
```diff
@@ -105,3 +105,9 @@ statuspanel:-moz-locale-dir(ltr):not([mirror]) .statuspanel-inner {
 #verticaltabs-box .private-browsing-indicator {
   display: none;
 }
+
+#TabsToolbar[brighttext] .tabs-newtab-button,
+#TabsToolbar[brighttext] > #new-tab-button,
+#TabsToolbar[brighttext] > toolbarpaletteitem > #new-tab-button {
```
Comment (bwinton) [inline skin/base.css]: Does this also apply to the dark DevTools theme?  Cause if so, we should change line 94 for consistency, and if not, we should add `[devtoolstheme="dark"]` to these…

Comment (johngruen) [inline skin/base.css]: Nope!

**Comment by johngruen** [comment]
screen shot 2016-04-26 at 2 38 51 pm
screen shot 2016-04-26 at 2 38 46 pm
screen shot 2016-04-26 at 2 38 33 pm
screen shot 2016-04-26 at 2 38 12 pm

**Comment by johngruen** [comment]
@bwinton the ESPN tab is b/c i was testing audio :)

**Pull request merged**